bin/wpc_init

```diff
@@ -121,7 +134,22 @@ set -e
 ## This code will be run during setup, INSIDE the container.
 ##
 
-wp core install --skip-email --admin_user=admin --admin_password=admin --admin_email=admin@localhost.invalid --url=http://localhost --title=Site
-# wp theme activate ...
-# wp plugin activate ...
+##############
+# Config
+##############
+title=your_site_title
```

Comment:
@tomdxw Sorry, it needed quotes in the internal.sh script as well - should be fixed now.